deploy/kubernetes/nexentastor-csi-driver-block-config.yaml

```diff
@@ -13,7 +13,7 @@ nexentastor_map:
     defaultVolumeGroup: csiDriverPool/csiVolumeGroup            # default volume group for driver's volumes [pool/volumeGroup]
     defaultTargetGroup: tg1                                     # [required] NexentaStor iSCSI target group name
     defaultTarget: iqn.2005-07.com.nexenta:01:test              # [required] NexentaStor iSCSI target
-    defaultHostGroup: all                                       # [required] NexentaStor host group
+    # defaultHostGroup: all                                       # [required] NexentaStor host group
 
```

Comment:
This line is commented out, but is mentioned as [required], please clarify this

Reply:
right, this is not required any more, will remove